### DISCUSSION TITLE
Assinatura Digital de Códigos

### DISCUSSION THREAD

**Discussion by wgabrjir** [community/community#145481] [Programming Help]
### Body

Estou com problema para distribuir meus arquivos (.exe)

Apesar de todos os cuidados em baixar Bibliotecas e etc, sempre informa que há Vírus no (.exe).
Pesquisando , vi que é importante fazer assinatura digital destes arquivos.
Qual AC (Autoridade Certificadora) você utiliza ? É no Brasil ?
O Procedimento utilizando o Windows SDK , já entendi perfeitamente.
Só falta para mim o certificado Digital.
Tenho Certificado Digital (E-CNPJ) para documentos da minha Empresa.
Mas esta não funciona para Códigos.
Gostaria de algumas sugestões.

Obrigado

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mecodeatlas** [comment]
Hi @wgabrjir , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
